opentelemetry-semantic-conventions/src/opentelemetry/semconv/v1_23_1/Http_metrics.py

```diff
@@ -0,0 +1,123 @@
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry.metrics import (
+    Counter,
+    Histogram,
+    Meter,
+    UpDownCounter,
+    ObservableGauge,
+)
+from typing import Callable, Sequence
+
+class HttpMetrics:
+
+    """
+    Size of HTTP client request bodies
+    """
+    @staticmethod
+  
+    def create_http_client_request_body_size(meter: Meter) -> Histogram:
+  
+        return meter.create_histogram(
+            name="http.client.request.body.size",
+            description="Size of HTTP client request bodies.",
+            unit="By",
+        )
+
+
+    """
+    Duration of HTTP client requests
+    """
+    @staticmethod
+  
+    def create_http_client_request_duration(meter: Meter) -> Histogram:
```

Comment:
Would you  like something like this vs 

```python
    HTTP_SERVER_REQUEST_DURATION = "http.server.request.duration"
    """
    Measures the duration of inbound HTTP requests
    Instrument: histogram
    Unit: s
    """
```

Reply:
resolving in favor of https://github.com/open-telemetry/opentelemetry-python/pull/3586#discussion_r1509494281